quickcomm/views.py

```diff
@@ -159,15 +162,34 @@ def post_like(request, post_id, author_id):
     # post = get_object_or_404(Post, pk=post_id)
 
     if request.user.is_authenticated:
-        author = Author.objects.all().get(user=request.user)
-        like_obj, created_obj = Like.objects.get_or_create(post=post, author=author)
-        if not created_obj:
-            like_obj.delete()
-        like_key = f"post_like_{post_id}"
-        request.session[like_key] = not created_obj
-
+        if request.method == 'POST':
+            author = Author.objects.all().get(user=request.user)
+            like_obj, created_obj = Like.objects.get_or_create(post=post, author=author)
+            if not created_obj:
+                post.likes.remove(request.user)
+                like_obj.delete()
+            else:
+                post.likes.add(request.user)
+            like_key = f"post_like_{post_id}"
+            request.session[like_key] = not created_obj
     return redirect("post_view", post_id=post_id, author_id=author_id)
 
+   
+@login_required
+def like_comment(request, post_id, author_id, comment_id):
```

Comment:
I just tried to like a comment and got a value error at authors/author_id/posts/post_id/like_comment because it's saying it didn't return an HttpResponse. It seems like the code does get to the redirect for like_comment, it's just not working for me for some reason. I just nuked my db and don't have any migrations. Any suggestions on what to try?